data.json

```diff
@@ -1,3782 +1,2928 @@
 [
```

Comment:
Sorting all the elements in this array ordered by "project" should show the diff much better, otherwise will be quite difficult to review

Reply:
Done, ordered the projects alphabetically